packages/@react-aria/table/intl/en-US.json

```diff
@@ -5,5 +5,7 @@
   "ascending": "ascending",
   "descending": "descending",
   "ascendingSort": "sorted by column {columnName} in ascending order",
-  "descendingSort": "sorted by column {columnName} in descending order"
+  "descendingSort": "sorted by column {columnName} in descending order",
+  "longPressToSelect": "Long press to enter selection mode.",
+  "cmdPressToSelect": "Command press to add to selection."
```

Comment:
are these strings ok?
do we need ones like opt + arrow navigates without replacing selection?